test/form-builder/TextField.test.tsx

```diff
@@ -1,17 +1,83 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-empty-function */
+
 import React from 'react';
 import TextField from '../../src/form-builder/TextField';
+import { waitFor } from '@testing-library/react';
+
+import { buildRenderForm, changeValue } from '../utils';
 
-import { buildRenderForm } from '../utils';
+const renderForm = buildRenderForm();
 
-const renderForm = buildRenderForm({ name: 'asdf' });
+const getInput = (container: HTMLElement) => {
+  const input = container.querySelector('va-text-input') as HTMLInputElement;
+  if (!input) throw new Error('No va-text-input found');
+  return input;
+};
 
 describe('Form Builder - TextField', () => {
   test('renders', () => {
-    const { getByLabelText } = renderForm(
+    const { container } = renderForm(
+      <TextField name="thing" label="The Thing" />
+    );
+    const input = getInput(container);
+    expect(input.getAttribute('label')).toEqual('The Thing');
+    expect(input.getAttribute('name')).toEqual('thing');
+  });
+
+  test('renders initial value', () => {
+    const rf = buildRenderForm({ thing: 'asdf' });
+    const { container } = rf(<TextField name="thing" label="The Thing" />);
+    const input = getInput(container);
+    expect(input.getAttribute('value')).toEqual('asdf');
+  });
```

Comment:
Is it a Formik thing that prevents us from setting the `value` in the JSX to-be-rendered? This structure may take some getting used to.

Reply:
Yes. The value lives in the `Formik` state. That's the source of truth.